src/Ng/Jobs/Catalog2LuceneJob.cs

```diff
@@ -107,6 +107,13 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
             }
         }
 
+        ReadCursor GetTheLeastAdvancedRegistrationCursor(string registrationArg, CancellationToken cancellationToken)
+        {
+            string[] registrations = registrationArg.Split(';');
```

Comment:
Any reason we can't just query index.json and get all of the registrations from there?